test/graphgym/test_graphgym.py

```diff
@@ -170,6 +170,9 @@ def destroy_process_group():
         torch.distributed.destroy_process_group()
 
 
+ggg = 0
```

Comment:
whats this for? can we remove it? otherwise lgtm

Reply:
Done